utils/src/utils/AzExtFsExtra.ts

```diff
@@ -76,6 +76,16 @@ export namespace AzExtFsExtra {
         return !!stats;
     }
 
+    export async function readJSON(resource: Uri | string): Promise<unknown> {
```

Comment:
nit: Would you want to use a generic instead of `unknown` here?

Reply:
Agreed, I like generic better.